NEWS.md

```diff
@@ -97,6 +97,8 @@ function calls. (#850, #851, @renkun-ken)
 * `assignment_linter()` now lints right assignment (`->` and `->>`) and gains two arguments. `allow_cascading_assign` (`TRUE` by default) toggles whether to lint `<<-` and `->>`; `allow_right_assign` toggles whether to lint `->` and `->>` (#915, @michaelchirico)
 * `infix_spaces_linter()` gains argument `exclude_operators` to disable lints on selected infix operators. By default, all "low-precedence" operators throw lints; see `?infix_spaces_linter` for an enumeration of these. (#914 @michaelchirico)
 * `infix_spaces_linter()` now throws a lint on `a~b` and `function(a=1) {}` (#930, @michaelchirico)
+* `object_length_linter()` correctly detects generics and only counts the implementation class towards the length. 
+  (#871, @AshesITR)
```

Comment:
An example here would be nice -- I needed to read the tests to understand what this meant